extras/schemas/resin-os.json

```diff
@@ -0,0 +1,185 @@
+{
+    "config_txt": {
+        "type": "json",
+        "location": { 
+            "partition": 1,
+            "path": ["config.json"]
+        },
+        "properties": [
+            {
+                "definition": {
+                    "persistentLogging": {
+                        "type": ["boolean"],
+                        "mapping": ["/persistentLogging"],
+                        "optional": true
+                    },
+                    "hostname": {
+                        "type": ["string"],
+                        "mapping": ["/hostname"],
+                        "optional": true
+                    },
+                    "application": {
+                        "properties": [
+                            {
+                                "definition": {
+                                    "name": {
+                                        "type": ["string"],
+                                        "mapping": ["/applicationName"]
+                                    },
+                                    "id": {
+                                        "type": ["string"],
+                                        "mapping": ["/applicationId"]
+                                    }
+                                }
+                            }
+                        ]
+                    },
+                    "deviceType": {
+                        "type": ["string"],
+                        "mapping": ["/deviceType"]
+                    },
+                    "userId": {
+                        "type": ["string"],
+                        "mapping": ["/userId"]
+                    },
+                    "appUpdagePollInterval": {
```

Comment:
It would be nice to use the "description" attribute to add more context and additional information (e.g. here we could add more info about the value that's expected here (is it seconds, milliseconds?)

Reply:
In the current implementation, there is no description field. The schema format is going to change significantly in the next version.